### PR TITLE
Version Bump to 1.0.26

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ Please format the changes as follows:
 + BugFixes:
 + Updates:
 
+# 1.0.26
+
 # 1.0.25
+This version did not get published due to issues encountered in App Service. Mitigation was taken by repackaging the previously released version (1.0.21) as 1.0.25.
```
Comment (jander-msft) [inline CHANGELOG.md]: nit: space before "The"

Comment (WilliamXieMSFT) [reply CHANGELOG.md]: I'm not quite sure what you mean, the spacing looks fine to me?

Comment (jander-msft) [reply CHANGELOG.md]: Huh, that's odd. When on the conversation tab, it shows that it has a space, but when on the file changes tab, it does not.

**Pull request merged**